lib/views/pages/events/events.dart

```diff
@@ -162,7 +161,8 @@ class _EventsState extends State<Events> {
     final String mutation = Queries().deleteEvent(eventId);
     final Map result = await apiFunctions.gqlquery(mutation);
     if (result["exception"] != null) {
-      _exceptionToast("Could not delete event! Please try again later");
+      CustomToast.exceptionToast(
```

Comment:
Does the type of exception matter here? If it's internet connectivity i.e. ClientException as you mentioned earlier then we should let them know that's the issue. Otherwise we show some other message if there is something they can do to resolve it. If nothing else, then generic error - maybe ask them to recreate it.

Reply:
> Does the type of exception matter here? If it's internet connectivity i.e. ClientException as you mentioned earlier then we should let them know that's the issue. Otherwise we show some other message if there is something they can do to resolve it. If nothing else, then generic error - maybe ask them to recreate it.

Oh yes, there's actually no need to show: `Could not delete event! Please try again later`
instead, we can show whether it's a client exception or server error.
Creating a commit for the same.
Thanks for pointing it out @dellyjm :)